tests/core/test_exceptions_longjmp4.cpp

```diff
@@ -0,0 +1,27 @@
+#include <stdio.h>
+#include <setjmp.h>
+
+static jmp_buf buf;
+
+void foo() {
+  throw 3;
+}
+
+int main() {
+  int jmpval = setjmp(buf);
+  if (jmpval != 0) {
+    printf("setjmp returned for the second time\n");
```

Comment:
No it's not gonna be reached; as well with `printf("inner catch: caught %d\n");`. These tests check if compilation of mixed `setjmp` and `try`-`catch` work because LLVM backend does heavy transformation on functions that has `setjmp` calls, so I think it's fine that this line is not reached.

Reply:
Ok, maybe add a comment then, that this just checks compilation, and it is never reached?

Reply:
Done